### ISSUE TITLE
Bintray down

### ISSUE THREAD

**Issue by MoeLa** [jmini/Eclipse-Copyright-Generator#14]
Is there another Update-URL for Eclipse-Copyright-Generator than https://dl.bintray.com/jmini/Eclipse-Copyright-Generator/ ?

**Comment by skarpushin** [comment]
Hi @MoeLa have you found a solution?

**Comment by MoeLa** [comment]
Nope, I used another tool => https://wiki.eclipse.org/Development_Resources/How_to_Use_Eclipse_Copyright_Tool

**Comment by skarpushin** [comment]
@MoeLa I was not able to install this either in my latest STS.

But I was able to solve the task by using http://jautodoc.sourceforge.net/. 

Also it turns out that there is https://www.mojohaus.org/license-maven-plugin/ which might be even more convenient from the operations perspective.

**Comment by jerocobo** [comment]
So... no one is maintaining this project?

**Comment by jreed-cartago** [comment]
> So... no one is maintaining this project?

It doesn't look as if it is supported. 

~~I'm using the tool @MoeLa posted above. The trick here is not to update the version number to the URL to your current Eclipse version, there doesn't seem to be any object behind it. The 4.19 version works in my eclipse as of 2022-03 (4.23.0)~~
Nope it isn't working on 2022-03 either, and caused lots of issues. 

Finally got my problem solved as @skarpushin with the JAutodoc. It has the ability to create headers as well as JavaDoc. You can actually create only headers without the JavaDoc. This works and supports current versions of Eclipse.
